AUTHORS

```diff
@@ -12,6 +12,7 @@ Alexander Egurnov <alexander.egurnov@gmail.com>
 Andrei Blinnikov <goofinator@mail.ru>
 Bill Gray <wgray@gogray.com>
 Bill Noon <noon.bill@gmail.com>
+Brandon Dube <brandon.dube@jpl.nasa.gov> <brandon@retrorefractions.com>
```

Comment:
I'm going to squash this PR, so we will lose the commit message. Please revert this and [make a new PR making this change](https://github.com/gonum/gonum/pull/1437#pullrequestreview-483727703).

Reply:
ahead of you in parallel -- sorry, I'm too accustomed to single branch or only two branch development.

Reply:
No worries. Yes, GitHub's PR mechanism is not the best.